### PR TITLE
feat: picker component

### PR DESCRIPTION
**Pull request by guillaumewttj**

### CODE REVIEW

**Comment by theo-mesnil** [comment]
Rename Picker to IconColorPicker :)

File: packages/Picker/doc.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+name: Picker
+route: /fields/picker
+menu: Fields
+---
+
+import { Props, Playground } from 'docz'
+import { ConnectedField } from '../ConnectedField'
```
Comment (theo-mesnil) [inline packages/Picker/doc.mdx]: You already have this in global :)

File: packages/Picker/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    const radio = useRadioState({ state: value })
+
+    const handleClick = e => {
+      e.stopPropagation()
+      radio.onClick && radio.onClick(e)
+      onChange && onChange(e)
+    }
+
+    return (
+      <FieldGroup as={ReakitRadioGroup} mb={0} ref={ref} required={required}>
```
Comment (theo-mesnil) [inline packages/Picker/index.js]: add the possibility to add styled system props on this components
```suggestion
      <FieldGroup as={ReakitRadioGroup} mb={0} ref={ref} required={required} {...rest}>
```

File: packages/Picker/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import { array, bool, func, string } from 'prop-types'
+import { Box } from '@xstyled/styled-components'
+import { RadioGroup as ReakitRadioGroup, useRadioState } from 'reakit/Radio'
+import { FieldGroup } from '@welcome-ui/field-group'
+import { Label } from '@welcome-ui/label'
+
+import * as S from './styles'
+
+export const Picker = forwardRef(
+  ({ connected, label, name, onChange, options, required, value }, ref) => {
```
Comment (theo-mesnil) [inline packages/Picker/index.js]:
```suggestion
  ({ connected, label, name, onChange, options, required, value, ...rest }, ref) => {
```

**Comment by guillaumewttj** [comment]
[Forms : Color and Icon picker > WUI](https://trello.com/c/cD7v6W2m/104-forms-color-and-icon-picker-wui)

File: packages/Picker/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# @welcome-ui/picker
+
+The [InputText](http://welcome-ui.com/fields/picker) component from [@welcome-ui](http://welcome-ui.com).
```
Comment (anucreative) [inline packages/Picker/README.md]: `The [Picker](http://welcome-ui.com/fields/picker)`…

File: packages/Picker/index.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+)
+
+Picker.displayName = 'Picker'
+
+Picker.propTypes /* remove-proptypes */ = {
+  connected: bool,
+  label: string,
+  name: string.isRequired,
+  onChange: func,
+  options: array.isRequired,
```
Comment (P1X3L) [inline packages/Picker/index.js]:
```suggestion
  options: arrayOf(
    shape({
        element: oneOf([string, function]),
        value: bool
    })
  )
```

**Pull request merged**